components/WebServer/apidoc/api/conversations/conversation.json

```diff
@@ -398,5 +398,68 @@
         "$ref": "#/components/responses/Unauthorized"
       }
     }
+  },
+  "/api/conversations/{conversationId}/media": {
+    "get": {
+      "tags": [
+        "conversations"
+      ],
+      "summary": "Get media from a conversation",
+      "description":"Set response type to \"audio/mpeg\" to get the mp3 file, or to \"application/json\" to get audiowavesurfer json file.",
+      "parameters": [{
+        "in": "path",
+        "name": "conversationId",
+        "type": "string",
+        "format": "uuid",
+        "required": true,
+        "description": "uuid string identifier for an conversation "
+      }
+      ],
+      "responses": {
+        "200": {
+          "description": "OK",
+            "content": {
+              "audio/mpeg": {
+                "type": "file"
+              },
+              "application/json": {
```

Comment:
It would be probably better to create a schemas for the format (with an truncated example)